components/ledger/cmd/script_test.go

```diff
@@ -0,0 +1,96 @@
+package cmd
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/formancehq/stack/libs/go-libs/api"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/uuid"
+	"github.com/numary/ledger/pkg/api/controllers"
+	"github.com/numary/ledger/pkg/core"
+	"github.com/spf13/viper"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_ScriptCommands(t *testing.T) {
+	ledger := uuid.NewString()
+	viper.Set("name", ledger)
+	_ = NewStorageInit().Execute()
+
+	d1 := []byte(`
+		send [EUR 1] (
+			source = @world
+			destination = @alice
+		)`)
+	path := os.TempDir() + "/script"
```

Comment:
Use filepath.Join instead of hardcoded "/" (again, for windows testing)

Reply:
done